### DISCUSSION TITLE
Royal Mail Default for the UK

### DISCUSSION THREAD

**Discussion by appsfinder2** [moralmunky/Home-Assistant-Mail-And-Packages#905] [General]
is there anyway to change default mail to Royal Mail instead of USPS i.e what code do I need to change and what files need to be updated please advise thanks.

**Comment by firstof9** [comment]
There is no "default", you choose your carrier from the list.

**Reply by appsfinder2** [reply]
> There is no "default", you choose your carrier from the list.
![Overview_–_Home_Assistant](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/assets/31155587/1db67594-5e01-44cc-aa5a-2c127eef54ca)

**Reply by firstof9** [reply]
That's the card, you want [the card repo](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages-Custom-Card).

**Comment by appsfinder2** [comment]
ive got the card installed there no option for Royal Mail has you can see from above I can put the royal mail sensor in where it says usps but then I does not work please update card for all postal services or am I missing something
